inst/golem-config.yml

```diff
@@ -5,4 +5,4 @@ default:
 production:
   app_prod: yes
 dev:
-  golem_wd: !expr here::here()
+  golem_wd: !expr golem::pkg_path()
```

Comment:
@JamesBisese - I can fix/restore the unintentional changes to app.R and golem-config.yml. Once you get a chance to update the provider filter settings, I will merge this pull request. Thanks!

Reply:
This is the GUI element I have on my computer. It uses the metaphor of having 3 radio buttons - USGS, WQX, and Both.

![image](https://github.com/user-attachments/assets/3f93a3bf-49a1-43eb-847a-e52d22c69344)

Reply:
The original check boxes work fine. In a new PR (https://github.com/USEPA/TADAShiny/pull/165), I made it so both are checked by default: 

<img width="83" alt="image" src="https://github.com/user-attachments/assets/461b9fe4-42a4-46fa-92ff-4dcd206b2863">